lib/oli_web/live/sections/overview_view.ex

```diff
@@ -487,13 +496,15 @@ defmodule OliWeb.Sections.OverviewView do
   end
 
   def handle_event("save_prompt", _, socket) do
-
     section = socket.assigns.section
 
-    Oli.Delivery.Sections.update_section(section, %{page_prompt_template: socket.assigns.page_prompt_template})
+    Oli.Delivery.Sections.update_section(section, %{
+      page_prompt_template: socket.assigns.page_prompt_template
+    })
 
-    socket = socket
-    |> put_flash(:info, "Prompt successfully saved")
+    socket =
+      socket
+      |> put_flash(:info, "Prompt successfully saved")
```

Comment:
IMO it's also more readable to avoid pipes when calling only one function.
I'd do `put_flash(socket, :info, "Prompt successfully saved")` instead.

[Reference](https://hexdocs.pm/credo/Credo.Check.Readability.SinglePipe.html)